src/carts/src/main/java/mushop/carts/Main.java

```diff
@@ -3,8 +3,11 @@
 import io.helidon.config.Config;
 import io.helidon.health.HealthSupport;
 import io.helidon.health.checks.HealthChecks;
+import org.eclipse.microprofile.health.HealthCheck;
```

Comment:
mp?

Reply:
No. Not using MP, but these custom health checks are in the MP package, but made available through io.helidon.health:helidon-health jars, which seemingly don't need MP. 

https://helidon.io/docs/latest/#/health/01_health